src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -703,6 +718,12 @@ def load_arguments(self, _):
     with self.argument_context('functionapp create') as c:
         c.argument('vnet', options_list=['--vnet'], help="Name or resource ID of the regional virtual network. If there are multiple vnets of the same name across different resource groups, use vnet resource id to specify which vnet to use. If vnet name is used, by default, the vnet in the same resource group as the webapp will be used. Must be used with --subnet argument.")
         c.argument('subnet', options_list=['--subnet'], help="Name or resource ID of the pre-existing subnet to have the webapp join. The --vnet is argument also needed if specifying subnet by name.")
+        c.argument('environment', options_list=['--environment'], help="Name of the container app environment.", is_preview=True)
```

Comment:
The default option name does not need to be explicitly declared
```suggestion
        c.argument('environment', help="Name of the container app environment.", is_preview=True)
```